commit 2184f0a491ad947776ce9098ccc3ed341745ed3d
Author: Nigel Kukard <nkukard@lbsd.net>
Date:   Tue Mar 14 13:28:30 2023 +0000

    fix: fixed session clearing and let logout_user take care of cleanup

diff --git a/powerdnsadmin/routes/index.py b/powerdnsadmin/routes/index.py
index 72305a6..091f1f5 100644
--- a/powerdnsadmin/routes/index.py
+++ b/powerdnsadmin/routes/index.py
@@ -526,7 +526,6 @@ def clear_session():
     session.pop('google_token', None)
     session.pop('authentication_type', None)
     session.pop('remote_user', None)
-    session.clear()
     logout_user()
 
 
